.github/workflows/test.yml

```diff
@@ -4,9 +4,19 @@ on:
   push:
     branches:
     - main
+
 jobs:
+  lint: 
+    runs-on: ubuntu-latest
```

Comment:
I've thought about that too actually, why do you think it shouldn't be possible? We can read from a file in the workflow. The only issue would be conflicts with the config file in the repo. Maybe we can use the repo one when provided and use a central config file otherwise

Reply:
It's perfectly doable, yeah. I can se it up.